builtin/v8/market/market_params.go

```diff
@@ -0,0 +1,138 @@
+package market
+
+import (
+	addr "github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-bitfield"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/go-state-types/builtin/v8/util/adt"
+	"github.com/filecoin-project/go-state-types/exitcode"
+	cbg "github.com/whyrusleeping/cbor-gen"
+	"golang.org/x/xerrors"
+)
+
+type WithdrawBalanceParams struct {
+	ProviderOrClientAddress addr.Address
+	Amount                  abi.TokenAmount
+}
+
+type PublishStorageDealsParams struct {
+	Deals []ClientDealProposal
+}
+
+type PublishStorageDealsReturn struct {
+	IDs        []abi.DealID
+	ValidDeals bitfield.BitField
+}
+
+// - Array of sectors rather than just one
```

Comment:
It looks like you dropped the first line of this comment ("changes since version xxx")